packages/backend/src/handlers/actions/quests/createQuest/permissions.ts

```diff
@@ -0,0 +1,22 @@
+import { numbers } from '@metafam/utils';
+
+import { getERC20Contract } from '../../../../lib/ethereum';
+
+const { BN, amountToDecimal } = numbers;
+
+/**
+ * As a first iteration, we only allow people to create quests if they hold more that 100 pSEED tokens
+ */
+
+export async function isAllowedToCreateQuest(playerAddress: string): Promise<boolean> {
+  const pSEEDContractAddress = '0x34a01c0a95b0592cc818cd846c3cf285d6c85a31';
+  const pSEEDContract = getERC20Contract(pSEEDContractAddress);
```

Comment:
should extract this out into a separate file since we will likely by using it in other places. The contract adds should be in a config file somewhere.

Reply:
This is why I have put this function in a file of its own, but didn't move it far away and name it generically because I don't know yet how and where we would use that again, so I didn't take any step further. 

Same for contract address, this is very temporary, as it will check pSEED balance of caller for creation of any quest (even for quest in other guilds than metagame)

In later stages, the permission check should be customisable by guild, like [strategies in snapshot](https://docs.snapshot.page/strategies), so this behaviour or token address should be fetched from the guild.

If you prefer, I can put the pSEED address into a environment variable, but I don't want to create a generic system without knowing where we going exactly.